smart_open/s3.py

```diff
@@ -177,7 +186,8 @@ def seek(self, position):
             #
             self._body = io.BytesIO()
         else:
-            self._body = self._object.get(Range=range_string)['Body']
+            getkwargs = dict(Range=range_string)
+            self._body = _get(self._object, self._version_id, **getkwargs)['Body']
```

Comment:
This is more complicated than it needs to be.

```suggestion
            self._body = _get(self._object, self._version_id, Range=range_string)['Body']
```

The line above is not necessary.